stream_alert_cli/terraform/kinesis_streams.py

```diff
@@ -30,6 +30,12 @@ def generate_kinesis_streams(cluster_name, cluster_dict, config):
     prefix = config['global']['account']['prefix']
     config_modules = config['clusters'][cluster_name]['modules']
 
+    infrastructure_config = config['global'].get('infrastructure')
+    shard_level_metrics = []
+    if infrastructure_config and 'monitoring' in infrastructure_config:
+        if 'shard_level_metrics' in infrastructure_config['monitoring']:
```

Comment:
Check out the [get](https://docs.python.org/2/library/stdtypes.html#dict.get) method on a dictionary